wiki/en/en-Installation-for-Linux.md

```diff
@@ -14,18 +14,20 @@ Although we don't provide official packages for every Linux distribution, you mi
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/jamulus.svg)](https://repology.org/project/jamulus/versions)
 
-There is also list of unofficial [packages for different distributions here](https://github.com/corrados/jamulus/issues/223).
+There is also list of unofficial [packages for different distributions here](https://github.com/jamulussoftware/jamulus/discussions/914).
 
-### Debian and Ubuntu (amd64 .deb only)
+### Debian and Ubuntu
+
+Jamulus (with GUI) is will be [included in Debian Bullseye (testing)](https://packages.debian.org/bullseye/jamulus) thanks to [mirabilos](https://github.com/mirabilos). If you already use Bullseye, just install Jamulus via `sudo apt-get install jamulus`.
 
 If you're on amd64 **Debian**/**Ubuntu**, you may try the compiled .deb packages from GitHub Actions.
 
-1. Download the jamulus_latest_ubuntu_amd64.deb file from the [Jamulus GitHub release](https://github.com/corrados/jamulus/releases/tag/latest) page
+1. Download the jamulus_latest_ubuntu_amd64.deb file from the [Jamulus GitHub release](https://github.com/jamulussoftware/jamulus/releases/tag/latest) page
```

Comment:
I thought we were in the Ubuntu repos?

Reply:
No. We're in the Debian repos. Maybe this also adds us to the Ubuntu repo automatically?

Reply:
Hm. I get "Unable to locate package jamulus" when I do apt install jamulus on Debian 10 anyway...

Reply:
Oh wait it's only for Bullseye. OK. Very few people will be using that :-)

Reply:
Bullseye will (probably) be the new Debian "stable" in a few months now.
Maybe it worth to put a warning on this while Bullseye isn't yet the "stable" version of Debian?

Reply:
Bullseye doesn't have a [release date](https://wiki.debian.org/DebianReleases) listed yet, and I don't know when the Ubuntu repos will incorporate Bullseye software. So for most Linux users apt install isn't going to be a thing for quite while maybe. 

Also BTW I note [somebody saying](https://sourceforge.net/p/llcon/discussion/software/thread/aa3960d2e7/#08d4) that the dependencies for Fedora were wrong in the docs

Reply:
> Bullseye doesn't have a [release date](https://wiki.debian.org/DebianReleases) listed yet, and I don't know when the Ubuntu repos will incorporate Bullseye software.

I know that. The purpose of my previous message was to give an order of magnitude.

Debian policy on this matter is "it's going to be released when it'll be released", so waiting for a release date will not help because it'll not be announced before D-day. That said, [the release process has already been started](https://release.debian.org/bullseye/freeze_policy.html) and, at this time (and without any warranty, this is a moving target), it's fair to think that we __should__ have a Debian release around spring time / beginning of summer. Of course, something can always go wrong and the release would then be postponed.

About Ubuntu, that will depend what they will import from testing/sid, and when.

All of that to try to clarify that the purpose of my previous mail was: if we are to write that debian/ubuntu users can "apt install jamulus", it'd be wise to add a note on it that it'll be working for people using [Debian Bulleyes](https://packages.debian.org/search?keywords=jamulus) or [Ubuntu Hirsute](https://packages.ubuntu.com/search?keywords=jamulus), not others for the moment.

Reply:
For the time being, how easy would it be to automatically maintain an Ubuntu PPA?

Reply:
FYI, I tried to compile/build the Jamulus deb from the Bullseye source package (this is usually trivial) on a Debian 10 buster.
There is an error:

```
RCC: Error in 'src/resources.qrc': Cannot find file 'res/translation/translation_de_DE.qm'
make[1]: *** [Makefile:1629: src/res/qrc_resources.cpp] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/home/gerald/sources/debian/jamulus-3.6.2+dfsg1'
dh_auto_build: error: make -j4 returned exit code 2
make: *** [debian/rules:45: binary] Error 25
dpkg-buildpackage: erreur: debian/rules binary subprocess returned exit status 2
debuild: fatal error at line 1182:
dpkg-buildpackage -us -uc -ui -b failed
```
The files on `src/res/translation` have a `*.ts` extension, not `*.qm`. The `*.qm`files aren't build.
 
Compiling and installing Jamulus on a Linux Debian based (Raspberry Pi OS Buster, 32 and 64 bits) following the instructions on install linux page works fine (for the client, and for the server).
I've installed Jamulus 3.6.2 (and upgraded from 3.6.0 and 3.6.1) on a Raspberry Pi OS 32 bits (server), on a Raspberry Pi OS 64 bits (client), both are a Debian 10 buster based system, and a Ubuntu 20.04 following the compilation instructions on the linux page.

My opinion is to say:
- if an official distribution package exists, install the package of the distribution according to the usual procedures on it.
- if not, compile and install Jamulus by following step by step the instructions on the Linux installation page.

Installing testing package on a stable Ubuntu or Debian is not recommended!

Reply:
> this is usually trivial

I disagree. Debian 10 (buster) and Debian 11 (Bulleyes) are 2 different distributions with 2 different toolchain and 2 different packages libraries versions. It's not that unusual that a simple package rebuild fails.


> My opinion is to say:
>  if an official distribution package exists, install the package of the distribution according to the usual procedures on it.
>  if not, compile and install Jamulus by following step by step the instructions on the Linux installation page.

Agreed.


> Installing testing package on a stable Ubuntu or Debian is not recommended!

It's a general Debian/Ubuntu advice. It looks to be outside of the Jamulus's project jurisdiction.

Reply:
> > this is usually trivial
> 
> I disagree. Debian 10 (buster) and Debian 11 (Bulleyes) are 2 different distributions with 2 different toolchain and 2 different packages libraries versions. It's not that unusual that a simple package rebuild fails.

I have recompiled a lot of packages not available yet in Buster but available in testing from source.
It is really usualy trivial (it should be the case for jamulus) if all the deps build are available on Buster. And that is the case for Jamulus, everything needed to compile/build Jamulus on this system is available (as listed on the actual install linux page for the 3.6.2).

You have just to add the debian testing source (and only the source) to sources.list

```
# Uncomment deb-src lines below then 'apt-get update' to enable 'apt-get source'
deb-src http://deb.debian.org/debian buster main contrib non-free
deb-src http://deb.debian.org/debian-security/ buster/updates main contrib non-free
deb-src http://deb.debian.org/debian buster-updates main contrib non-free
# sources from testing/unstable 
deb-src http://httpredir.debian.org/debian unstable main
```

And then (for jamulus):

```
apt-get source jamulus
cd jamulus-3.6.2+dfsg1/
sudo apt-get build-dep jamulus # this will download and install the build deps from Buster packages
debuild -b -uc -us
cd ../
sudo apt install ./jamulus_3.6.2+dfsg1.arm64.deb # for example
```
I prefer using apt install instead of dpkg -i because apt install all the deps in one step.

What is very strange is that on this machine compiling from source works, not building the deb.
Something seems to be broken or missing in the deb source for the translations files.

>>  Installing testing package on a stable Ubuntu or Debian is not recommended!
> It's a general Debian/Ubuntu advice. It looks to be outside of the Jamulus's project jurisdiction.

Yes but adding testing or unstable repo in sources.list to install Jamulus with apt on a stable Debian or Ubuntu is not a good idea.

It is better (and it works fine following the doc) to install from sources if no official distribution (or team) package available. (IMHO)

Reply:
> I have recompiled a lot of packages not available yet in Buster but available in testing from source.

It **can** work but it's not to be recommanded to the end users in any way.

(BTW, I know how a backport works)

Reply:
I think a backport is already in preparation: https://github.com/jamulussoftware/jamulus/issues/896#issuecomment-767631624
let's just wait for this.

Reply:
Much better to tell the end users to use a package from buster-backports rather than backporting the package himself.